### PR TITLE
dev-libs/mpvqt: new package, add 1.0.0

### PR DESCRIPTION
**Pull request by redstrate**
MpvQt was finally released (yay!) and this will be needed by Tokodon in 9999/24.02, and also PlasmaTube 9999/24.02 (#933)

### CODE REVIEW

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+fi
+
+DESCRIPTION="MpvQt is a libmpv wrapper for QtQuick2 and QML"
+HOMEPAGE="https://invent.kde.org/libraries/mpvqt"
+
+LICENSE="|| ( GPL-2 GPL-3 LGPL-3 LGPL-2.1 ) CC-BY-SA-4.0 MIT BSD"
+SLOT="6"
+IUSE=""
+
+DEPEND="
```
Comment (thesamesam) [inline dev-libs/mpvqt/mpvqt-1.0.0.ebuild:22]: No RDEPEND looks wrong

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	media-video/mpv:=[libmpv]
+	>=dev-qt/qtdeclarative-${QTMIN}:6
+	>=kde-frameworks/extra-cmake-modules-${KFMIN}
```
Comment (thesamesam) [inline dev-libs/mpvqt/mpvqt-1.0.0.ebuild:25]: ecm should be a BDEPEND but eclass should handle this for you anyway

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+EAPI=8
+
+KFMIN=5.245.0
+QTMIN=6.5.0
+inherit ecm kde.org
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${PN}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
```
Comment (thesamesam) [inline dev-libs/mpvqt/mpvqt-1.0.0.ebuild:12]: drop all but ~amd64 (https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages)

**Review by thesamesam — CHANGES_REQUESTED** [review]

**Comment by redstrate** [comment]
I wonder if this should be under media-libs instead of dev-libs?

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild (hunk 3 above)
Comment (redstrate) [reply dev-libs/mpvqt/mpvqt-1.0.0.ebuild]: Fixed

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild (hunk 1 above)
Comment (redstrate) [reply dev-libs/mpvqt/mpvqt-1.0.0.ebuild:22]: Oops forgot that one, fixed too

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild (hunk 2 above)
Comment (redstrate) [reply dev-libs/mpvqt/mpvqt-1.0.0.ebuild]: D'oh right, fixed

**Comment by a17r** [comment]
> I wonder if this should be under media-libs instead of dev-libs?

Yes, suits better imo.

File: dev-libs/mpvqt/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="project">
+        <email>kde@gentoo.org</email>
+        <name>Gentoo KDE Project</name>
+    </maintainer>
+    <upstream>
+        <remote-id type="kde-invent">libraries/ktextaddons</remote-id>
```
Comment (a17r) [inline dev-libs/mpvqt/metadata.xml:9]: that looks like a leftover

File: dev-libs/mpvqt/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        <bugs-to>https://bugs.kde.org/</bugs-to>
+        <remote-id type="kde-invent">libraries/mpvqt</remote-id>
+    </upstream>
+    <use>
+        <flag name="designer">Build plugins for <pkg>dev-qt/designer</pkg></flag>
+        <flag name="speech">Enable text-to-speech support</flag>
+    </use>
+</pkgmetadata>
```
Comment (a17r) [inline dev-libs/mpvqt/metadata.xml:17]: missing newline

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild (hunk 1 above)
Comment (a17r) [reply dev-libs/mpvqt/mpvqt-1.0.0.ebuild:22]: I've checked ELF deps of /usr/lib64/libMpvQt.so and there are a couple required qtbase IUSE missing (ecm.eclass really only depends on dev-qt/qtbase to provide QtCore): `gui,opengl`

**Review by a17r — CHANGES_REQUESTED** [review]
Thanks!

File: dev-libs/mpvqt/mpvqt-1.0.0.ebuild (hunk 1 above)
Comment (redstrate) [reply dev-libs/mpvqt/mpvqt-1.0.0.ebuild:22]: Ah right, fixed

File: dev-libs/mpvqt/metadata.xml (hunk 4 above)
Comment (redstrate) [reply dev-libs/mpvqt/metadata.xml]: Oops, looks like that collided with something else I had locally. Removed

File: dev-libs/mpvqt/metadata.xml (hunk 5 above)
Comment (redstrate) [reply dev-libs/mpvqt/metadata.xml]: Fixed too

File: media-libs/mpvqt/mpvqt-1.0.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+KFMIN=5.245.0
+QTMIN=6.5.0
+inherit ecm kde.org
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${PN}/${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="MpvQt is a libmpv wrapper for QtQuick2 and QML"
```
Comment (a17r) [inline media-libs/mpvqt/mpvqt-1.0.0.ebuild:15]:
```suggestion
DESCRIPTION="libmpv wrapper for QtQuick2 and QML"
```

This one I will fix for you on merge.

**Review by a17r — APPROVED** [review]

**Pull request closed without merging**